a10_octavia/common/utils.py

```diff
@@ -385,45 +398,38 @@ def get_ipv6_address_from_conf(address_list, subnet_id):
                 final_address_list.append({'ipv6-addr': address[key]})
                 return final_address_list
 
+
 def get_ipv6_address(ifnum_oper, subnet, nics, address_list, loadbalancers_list):
     final_address_list = []
     network_driver = get_network_driver()
     amp_boot_networks = CONF.a10_controller_worker.amp_boot_network_list
     for nic in nics:
-        if nic.network_id in amp_boot_networks:
-            port_mac_address = network_driver.get_port(nic.port_id).mac_address.replace(":", "")
-            acos_mac_address = ifnum_oper['ethernet']['oper']['mac'].replace(".", "")
-            if port_mac_address == acos_mac_address:
+        port_mac_address = network_driver.get_port(nic.port_id).mac_address.replace(":", "")
+        acos_mac_address = ifnum_oper['ethernet']['oper']['mac'].replace(".", "")
```

Comment:
this is minor, but maybe
`acos_mac_address = ifnum_oper['ethernet']['oper']['mac'].replace(".", "")`
can move outside the `for nic in nics:` loop and don't need to run for every nic.

Reply:
DONE